pimd/pim_cmd.c

```diff
@@ -6294,13 +6304,39 @@ static void show_mroute_count(struct pim_instance *pim, struct vty *vty)
 		pim_inet4_dump("<source?>", c_oil->oil.mfcc_origin, source_str,
 			       sizeof(source_str));
 
-		vty_out(vty, "%-15s %-15s %-8llu %-7ld %-10ld %-7ld\n",
-			source_str, group_str, c_oil->cc.lastused / 100,
-			c_oil->cc.pktcnt - c_oil->cc.origpktcnt,
-			c_oil->cc.bytecnt - c_oil->cc.origbytecnt,
-			c_oil->cc.wrong_if - c_oil->cc.origwrong_if);
+		if (uj) {
+			json_object_object_get_ex(json, group_str, &json_group);
+
+			if (!json_group) {
+				json_group = json_object_new_object();
+				json_object_object_add(json, group_str,
+						       json_group);
+			}
+
+			json_source = json_object_new_object();
+			json_object_object_add(json_group, source_str,
+					       json_source);
+
+			json_object_int_add(json_source, "lastUsed",
+					    c_oil->cc.lastused / 100);
+			json_object_int_add(json_source, "Packets",
```

Comment:
should be the lower case first for single words

Reply:
Done.